lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -115,7 +135,9 @@ sub _add_routes {
         $r->name("$self->{route_prefix}$name") if $name;
       }
 
-      $self->_add_default_response($op_spec, $_) for @{$self->{default_response_codes}};
+      my $add_default_response_method = sprintf "_add_default_response_%s",
+        $self->validator->version;
+      $self->$add_default_response_method($op_spec, $_) for @{$self->{default_response_codes}};
```

Comment:
I don't think you need this loop, now that you also have the loop inside the methods.

Reply:
Yep, you right.
Fixed it.

Reply:
You're correct, new version resolves this issue.